acceptance/acceptance_test.go

```diff
@@ -2253,6 +2253,208 @@ include = [ "*.jar", "media/mountain.jpg", "/media/person.png", ]
 				})
 			})
 
+			when("creating asset cache", func() {
```

Comment:
There is definitely some simplification here, but I just want to push off doing a refactor of these tests because there will be many additions when testing asset-cache usage